### ISSUE TITLE
@get! deprecation warning

### ISSUE THREAD

**Issue by kcajf** [JuliaGizmos/WebIO.jl#426]
```
┌ Warning: `@get!(dict, key, default)` at /home/xxx/.julia/packages/WebIO/nTMDV/src/scope.jl:160 is deprecated, use `get!(()->default, dict, key)` instead.
└ @ Base deprecated.jl:204
┌ Warning: `@get!(dict, key, default)` at /home/xxx/.julia/packages/WebIO/nTMDV/src/scope.jl:357 is deprecated, use `get!(()->default, dict, key)` instead.
└ @ Base deprecated.jl:204
```

Seems like this has been deprecated in 1.5?

**Comment by travigd** [comment]
Yep. It's been silently discouraged for a while. This should be an easy fix.

**Comment by rasmushenningsson** [comment]
I've made a pull request #428 for this.

**Comment by PetrKryslUCSD** [comment]
I am on 0.8.14, I look at the code and it seems to have been fixed, yet I still see these warnings?

**Comment by rasmushenningsson** [comment]
> I am on 0.8.14, I look at the code and it seems to have been fixed, yet I still see these warnings?

The fix is only on the master branch so far. It will be fixed for everyone when 0.8.15 is released.

**Comment by rasmushenningsson** [comment]
This issue can be closed now that a new version has been released.
